pandas/_libs/tslibs/offsets.pyx

```diff
@@ -1373,6 +1373,24 @@ class DateOffset(RelativeDeltaOffset, metaclass=OffsetMeta):
         previous midnight.
     **kwds
         Temporal parameter that add to or replace the offset value.
+    weekday : int {0, 1, ..., 6}, default 0
+        A specific integer for the day of the week.
+        - 0 is Monday
+        - 1 is Tuesday
+        - 2 is Wednesday
+        - 3 is Thursday
+        - 4 is Friday
+        - 5 is Saturday
+        - 6 is Sunday.
```

Comment:
this doesn't look quite right, you need a blank line before "A specific integer"

![image](https://github.com/pandas-dev/pandas/assets/33491632/883f9348-235d-4f9c-b3b5-521ce64fc1fa)

Reply:
Working with @Detoro on this - we'd like to view the built HTML but have been encountering some issues.
Namely, after running `python make.py --single pandas.tseries.offsets.DateOffset` in the `doc` directory, we don't see any output in directory `reference/API` for DateOffset. We're getting this warning from Sphinx:

`/mnt/c/Users/batiu/OneDrive/Desktop/pandasContribution/pandas-toro/doc/source/index.rst:31: WARNING: autosummary references excluded document 'pandas.tseries.offsets.DateOffset'. Ignored.`

Are we building this class correctly, and if so why is the class's documentation being excluded by "autosummary references"?

Reply:
@MarcoGorelli PR is updated, can you take another look? We'll squash the commits as well.